palindrom-client.html

```diff
@@ -225,31 +209,38 @@
                 };
                 this.fire("generic-error", eventDetail);
             }
-
             onReconnectionCountdown(milliseconds) {
                 const eventDetail = {
                     milliseconds,
                     handled: false
                 };
-
                 this.fire("reconnection-countdown", eventDetail);
             }
-
             onReconnectionEnd() {
                 const eventDetail = {
                     handled: false
                 };
                 this.fire("reconnection-end", eventDetail);
             }
-
             onPatchApplied(patches, results) {
                 this.propagateDomBind({ patches, results });
                 this.fire("patch-applied", {
                     patches
                 });
             }
-
             notifyTemplateDomBind(tree, patchesAndResults, templateDomBind, polymerPathPrefix) {
+                if (patchesAndResults.patches.length > this.mergedNotificationsThreshold) {
+                    console.warn(`palindrom-client has noticed a huge patch of ${patchesAndResults.patches.length} operations, for better performance palindrom-client will replace the whole palindrom object instead of applying your updates individually`);
```

Comment:
I think that there should be a new sentence: `. For better performance`...

Also, the word `palindrom object` should be changed to `state object`.

Reply:
Ping. The second part of my comment was not addressed

Reply:
Addressed